app/dashboard/models.py

```diff
@@ -280,6 +280,7 @@ class Bounty(SuperModel):
         default=False, help_text=_('Admin override to mark as remarketing ready')
     )
     attached_job_description = models.URLField(blank=True, null=True)
+    event_tag = models.CharField(max_length=255, default='', blank=True)
```

Comment:
Would you be strongly opposed to moving this to a FK to Event? 

`event = models.ForeignKey('dashboard.HackathonEvent', related_name='bounties', null=True, on_delete=models.SET_NULL)`